operations/list.go

```diff
@@ -130,7 +130,7 @@ func listProjects(ctx context.Context, confPath string) error {
 		if prj.Id != prj.Identifier {
```

Comment:
Since we are switching the sort to be on the Identifier, I think it will make it more readable for users if we switch the columns to list identifier first.

Reply:
done